src/main/java/net/sf/jabref/remote/DBConnector.java

```diff
@@ -0,0 +1,110 @@
+/*  Copyright (C) 2003-2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.remote;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * Used to establish connections between JabRef and database systems like MySQL, PostgreSQL and Oracle.
+ */
+public class DBConnector {
+
+    private static final Log LOGGER = LogFactory.getLog(DBConnector.class);
+
+
+    /**
+     * Determines the suitable driver and retrieves a working SQL Connection in normal case.
+     * A default port is going to be taken.
+     *
+     * @param dbType Enum entry of {@link DBType} which determines the driver
+     * @param host Hostname, Domain or IP address
+     * @param database An already existent database name.
+     * @param user Username
+     * @param password Password
+     * @return
+     * @throws ClassNotFoundException Thrown if no suitable drivers were found
+     * @throws SQLException Thrown if connection has failed
+     */
+    public static Connection getNewConnection(DBType dbType, String host, String database, String user, String password)
+            throws ClassNotFoundException, SQLException {
+        return getNewConnection(dbType, host, getDefaultPort(dbType), database, user, password);
+    }
+
+    /**
+     * Determines the suitable driver and retrieves a working SQL Connection in normal case.
+     *
+     * @param dbType Enum entry of {@link DBType} which determines the driver
+     * @param host Hostname, Domain or IP address
+     * @param port Port number the server is listening on
+     * @param database An already existent database name.
+     * @param user Username
+     * @param password Password
+     * @return
+     * @throws ClassNotFoundException Thrown if no suitable drivers were found
+     * @throws SQLException Thrown if connection has failed
+     */
+    public static Connection getNewConnection(DBType dbType, String host, int port, String database, String user,
+            String password) throws ClassNotFoundException, SQLException {
+
+        String url = "jdbc:";
+
+        try {
+            if (dbType == DBType.MYSQL) {
```

Comment:
MultiQuery option is a thing needed for mysql, which allows multiple queries executed in one Statement. Otherwise it would throw an SQlException.
See here for details:
http://stackoverflow.com/questions/10797794/multiple-queries-executed-in-java-in-single-statement

While I see the need for the syntax differences, I would still suggest using the DBPreferences class for host/port/name and additional parameters. 
And if you look closely at the existing DbType enum, it contains a formatted Name, which is used for displaying the names in the DBImporter/Exporter combo boxes.